scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -435,3 +410,32 @@ check-rustdoc:
     - rusty-cachier snapshot create
     - time cargo +nightly doc --locked --workspace --all-features --verbose --no-deps
     - rusty-cachier cache upload
+
+.cargo-check-each-crate_template: &cargo-check-each-crate_template
+  stage:                           test
+  extends:
+    - .docker-env
+    - .test-refs
+  parallel: 2
```

Comment:
It will not work for a mac runner, we have only one currently

Reply:
I've removed `parallel` for Mac runners in the newer commits